### ISSUE TITLE
Could not make GPT2-Large transfer informal input to formal output

### ISSUE THREAD

**Issue by xszheng2020** [suzgunmirac/prompt-and-rerank#2]
Hi, @suzgunmirac Thanks for your work.

I am trying to reproduce the results in Table 22, but I found that GPT2-Large just repeats the informal input rather than transfers it. 

Did I mess up anything? Thanks!

`
outputs = model.generate(input_ids=input_ids, 
                         attention_mask=attention_mask,
                         num_beams=5, 
                         num_return_sequences=5, 
                         do_sample=False, 
                         eos_token_id=198,
                         max_length=len(input_ids[0])+64
                        )
`

> 
>  Here is a text, which is informal: {sorry but donnt know if i can do this alone.} Here is a rewrite of the text, which is formal: {I am sorry, but I don’t know if I can do this alone.} 
>  ### 
>  Here is a text, which is formal: {i am going to ask him to come to the concert with me, and i hope he accepts my invitation.} Here is a rewrite of the text, which is informal: {gonna ask him to come to the concert with me and hope he says yes :)} 
>  ### 
>  Here is a text, which is informal: {that sucks man but u gotta move on} Here is a rewrite of the text, which is formal: {that is unfortunate, but you need to move on} 
>  ### 
>  Here is a text, which is formal: {and i am sorry that you and your girlfriend broke up last week.} Here is a rewrite of the text, which is informal: {and im sorry that u and ur girlfriend broke up last week...}
>  ### 
> **Here is a text, which is informal: {follow ur heart... it's neva wrong} Here is a rewrite of the text, which is formal: {follow ur heart... it's neva wrong}**

**Comment by suzgunmirac** [comment]
Hi @xszheng2020, thank you so much for your interest in our work!

Overall, your model parameters look good to me, though you might want to be explicit about the `temperature` parameter just to be safe. 

In case it might be useful, I also created this notebook for you and other users. Please let me know if you have any further questions or comments. 

https://colab.research.google.com/drive/1kpO7-KHOv39_T58Oi0Z2fKsAJ6WXK0YY?usp=sharing

**Comment by xszheng2020** [comment]
Hi, @suzgunmirac Thank you so much!
I just tried the notebook and tuned the temperature.
GPT2-Large can output a better candidate now! 
However, it seems not very stable. Have to pick a good random seed...

["follow ur heart... it's neva wrong",
 "follow ur heart... it's neva wrong",
 "follow ur heart... it's neva wrong",
 **"follow your heart... it's neva wrong",**
 "follows ur heart... it's neva wrong",
 "follow ur heart... it's neva wrong",
 "follow ur heart... it's neva wrong...",
 "follow ur heart... it's a good thing u're not a girl with a heart disorder"]
